test/Elastic.Apm.AspNetCore.Tests/AspNetCoreMiddlewareTests.cs

```diff
@@ -88,10 +88,10 @@ public async Task HomeIndexSpanTest()
 			Assert.True(response.IsSuccessStatusCode);
 
 			var transaction = _capturedPayload.Payloads[0].Transactions[0];
-			Assert.NotEmpty(transaction.Spans);
+			Assert.NotEmpty(_capturedPayload.SpansOnFirstTransaction);

```

Comment:
Threw me a bit thinking it was part of the public API but its only part of `MockPayloadSender` in tests which is much nicer to read in tests 👍